src/SASjs.ts

```diff
@@ -595,7 +590,11 @@ export default class SASjs {
     const validationResult = this.validateInput(data)
 
     if (validationResult.status) {
-      if (config.serverType === ServerType.SasViya && config.contextName) {
+      if (
+        config.serverType !== ServerType.Sas9 &&
```

Comment:
will this line ever be true?

Reply:
The second part of this check `config.useComputeApi !== undefined && config.useComputeApi !== null` seems a bit unnecessary given that line 598 checks if `config.useComputeApi` is truthy.

Reply:
Also, should we not assume `useComputeApi` is false when the serverType is `SASVIYA` and `useComputeApi` has not been set to true or false?

Reply:
@krishna-acondy there is change in functionality that Allan required, there is a case where we want to use VIYA web approach WITH context name passed. So idea is this, useComputeApi: true | false | null/undefined (COMPUTE_API | JES_API | JES_WEB)

If it is SAS9 it will always be JES_WEB

Reply:
@allanbowe It will be true if:
```
useComputeApi: true or false
serverType: VIYA
```